tests/e2e/Makefile

```diff
@@ -33,8 +33,12 @@ help: ## Display this help.
 
 .PHONY: create_module_kubebuilder_project
 create_module_kubebuilder_project:
```

Comment:
The way this target is executed is `make create_module_kubebuilder_project`. I think it is misleading. Should we rename it to something like `make test-kubebuilder-module-creation`?